src/structures/ThreadChannel.js

```diff
@@ -419,6 +440,16 @@ class ThreadChannel extends Channel {
     return this.edit({ rateLimitPerUser }, reason);
   }
 
+  /**
+   * Set the applied tags for this channel (only applicable to forum threads)
+   * @param {GuildForumTag[]} appliedTags The tags to set for this channel
```

Comment:
```suggestion
   * @param {Snowflake[]} appliedTags The tags to set for this channel
```
as some pr says api takes id as it
also change typings 
or from my suggestion u can use somehow resolver

Reply:
![image](https://user-images.githubusercontent.com/71698422/191529473-497b5378-9925-4bc4-9db6-0279d3f7cb02.png)

Reply:
> ![image](https://user-images.githubusercontent.com/71698422/191529473-497b5378-9925-4bc4-9db6-0279d3f7cb02.png)

It is a array of object
but appliedTags takes ids
![image](https://user-images.githubusercontent.com/76836550/191530793-de1bfcc6-d79a-40f4-af64-b892c32fd8c1.png)

Reply:
> It is a array of objects but appliedTags takes ids ![image](https://user-images.githubusercontent.com/76836550/191530793-de1bfcc6-d79a-40f4-af64-b892c32fd8c1.png)

oh